_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -31966,13 +31966,13 @@
 /area/chapel/office)
 "bfW" = (
 /obj/structure/rack,
-/obj/item/storage/fancy/candle_box,
-/obj/item/storage/fancy/candle_box{
+/obj/item/storage/fancy/candle_box/full{
 	pixel_x = -3;
 	pixel_y = 3
 	},
-/obj/item/storage/fancy/candle_box{
-	pixel_x = 3;
+/obj/item/storage/fancy/candle_box/full,
+/obj/item/storage/fancy/candle_box/full{
+	pixel_x = -3;
```

Comment:
This one should be a 3 I believe

Reply:
I made it a -3 so they aren't stacked awkwardly on the shelf. Those values are typed in manually by me, not much difference to the original really.

Reply:
OK